MacrobenchmarkSample/app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,18 @@
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".ActivityLauncherActivity"
+            android:exported="true">
+            <intent-filter>
+                <action
+                    android:name="com.example.macrobenchmark.target.ACTIVITY_LAUNCHER_ACTIVITY" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </activity>
+        <!-- non-exported activities -->
```

Comment:
change comment:

Non exported activity, with no action can be launched internally, but not directly by a macrobenchmark. See NonExportedActivityBenchmark for how to navigate there indirectly.